docker-compose.yml

```diff
@@ -1,50 +1,63 @@
-
+version: '3'
 services:
-    tempo:
-        image: grafana/tempo
-        extra_hosts: ['host.docker.internal:host-gateway']
-        command: [ "-config.file=/etc/tempo.yaml" ]
-        volumes:
-            - ./docker/tempo/tempo-local.yaml:/etc/tempo.yaml:ro
-            - ./tempo-data:/tmp/tempo
-        ports:
-            - "14268"  # jaeger ingest
-            - "9411:9411" # zipkin
-
-    loki:
-        image: grafana/loki
-        extra_hosts: ['host.docker.internal:host-gateway']
-        command: [ "-config.file=/etc/loki/local-config.yaml" ]
-        ports:
-            - "3100:3100"                                   # loki needs to be exposed so it receives logs
-        environment:
-            - JAEGER_AGENT_HOST=tempo
-            - JAEGER_ENDPOINT=http://tempo:14268/api/traces # send traces to Tempo
-            - JAEGER_SAMPLER_TYPE=const
-            - JAEGER_SAMPLER_PARAM=1
-
+#    collector:
+#        image: otel/opentelemetry-collector:0.103.1
+#        extra_hosts: ['host.docker.internal:host-gateway']
+#        volumes:
+#            - ./docker/otel-collector/otel-collector-config.yaml:/etc/otelcol-contrib/config.yaml
+#        ports:
+#            - 1888:1888 # pprof extension
+#            - 8888:8888 # Prometheus metrics exposed by the collector
+#            - 8889:8889 # Prometheus exporter metrics
+#            - 13133:13133 # health_check extension
+#            - 4318:4318 # OTLP http receiver (for metrics and spans)
+#            - 55679:55679 # zpages extension
     prometheus:
-        image: prom/prometheus
+        image: prom/prometheus:v2.53.0 # https://hub.docker.com/r/prom/prometheus
         extra_hosts: ['host.docker.internal:host-gateway']
         command:
             - --enable-feature=exemplar-storage
+            - --enable-feature=otlp-write-receiver
+            - --web.enable-remote-write-receiver
             - --config.file=/etc/prometheus/prometheus.yml
         volumes:
+            - prometheus:/prometheus
             - ./docker/prometheus/prometheus.yml:/etc/prometheus/prometheus.yml:ro
         ports:
-            - "9090:9090"
-
+            - 9090:9090
```

Comment:
According to the official Docker Compose docs:
>"HOST:CONTAINER should always be specified as a (quoted) string,
to avoid conflicts with yaml base-60 float."

See: https://docs.docker.com/compose/compose-file/05-services/#short-syntax-3
See: https://yaml.org/type/float.html